### PR TITLE
using clang-format on the whole repo

### PR DESCRIPTION
**Pull request by dogoepp**
Please have a look to check that it doesn't break anything.

### CODE REVIEW

File: src/limbo/bayes_opt/boptimizer.hpp [hunk 1]
```diff
@@ -30,7 +30,8 @@ namespace limbo {
             boost::parameter::optional<tag::initfun>,
             boost::parameter::optional<tag::acquifun>,
             boost::parameter::optional<tag::stopcrit>,
-            boost::parameter::optional<tag::modelfun>> boptimizer_signature;
+            boost::parameter::optional<tag::modelfun>>
+            boptimizer_signature;
```
Comment (costashatz) [inline src/limbo/bayes_opt/boptimizer.hpp]: Why would it be like this? Do we have column/line limits?

File: src/limbo/experimental/bayes_opt/bo_multi.hpp [hunk 2]
```diff
@@ -91,8 +91,8 @@ namespace limbo {
                 boost::parameter::optional<tag::initfun>,
                 boost::parameter::optional<tag::acquifun>,
                 boost::parameter::optional<tag::stopcrit>,
-                boost::parameter::optional<tag::modelfun>> bo_multi_signature;
-
+                boost::parameter::optional<tag::modelfun>>
+                bo_multi_signature;
```
Comment (costashatz) [inline src/limbo/experimental/bayes_opt/bo_multi.hpp]: Same as before...

File: src/limbo/bayes_opt/boptimizer.hpp (hunk 1 above)
Comment (dogoepp) [inline src/limbo/bayes_opt/boptimizer.hpp]: In the style file, there doesn't seem to be any such limit. My guess is that there is a default setting to limit the length of lines as much as possible.

Does it bother you ?

Comment (costashatz) [inline src/limbo/bayes_opt/boptimizer.hpp]: > In the style file, there doesn't seem to be any such limit. My guess is that there is a default setting to limit the length of lines as much as possible.
> 
> Does it bother you ?

Not really, but it is weird..

Comment (dogoepp) [inline src/limbo/bayes_opt/boptimizer.hpp]: I'm sorry, I don't fully control clang-format. It seems to have a life of its own on some aspects. Do you validate or request changes ?

**Comment by costashatz** [comment]
@dogoepp I would suggest to close this PR for the moment and wait for #96 to be merged to master first and then run clang format. This (#96 ) is a big PR and it would be better to run clang format after we have merged. What do you think?

**Comment by dogoepp** [comment]
OK, perfectly fine for me. Then you could do the clang-format thing yourself :)

**Pull request closed without merging**